extensions/sns/src/main.rs

```diff
@@ -38,42 +34,67 @@ pub struct SnsOpts {
 /// Subcommands of `dfx sns`
 #[derive(Parser)]
 enum SubCommand {
-    /// Subcommands for working with configuration.
+    /// Deploy an sns directly to a subnet, skipping the sns-wasms canister.
+    /// The SNS canisters remain controlled by the developer after deployment.
+    /// For use in tests only.
     #[command()]
-    Config(SnsConfigOpts),
-    /// Subcommand for creating an SNS.
+    DeployTestflight(DeployTestflightArgs),
+    /// Manage the config file where the initial sns parameters are set.
     #[command()]
-    Deploy(DeployOpts),
+    InitConfigFile(InitConfigFileArgs),
+    /// Make changes to canisters you own to prepare for SNS Decentralization
+    #[command()]
+    PrepareCanisters(PrepareCanistersArgs),
+    /// Submit an NNS proposal to create new SNS.
+    #[command()]
+    Propose(ProposeArgs),
+
     /// Subcommand for importing sns API definitions and canister IDs.
     #[command()]
     Import(SnsImportOpts),
     /// Subcommand for downloading SNS WASMs.
     #[command()]
     Download(SnsDownloadOpts),
-    /// Subcommand for preparing dapp canister(s) for 1-proposal SNS creation
-    #[command()]
-    PrepareCanisters(SnsPrepareCanistersOpts),
-    /// Subcommand for submitting a CreateServiceNervousSystem NNS Proposal.
-    #[command()]
-    Propose(SnsProposeOpts),
 }
 
 /// Executes `dfx sns` and its subcommands.
 fn main() -> anyhow::Result<()> {
     let opts = SnsOpts::parse();
-    let dfx_cache_path = &opts.dfx_cache_path.ok_or_else(|| {
-        anyhow::Error::msg(
-            "Missing path to dfx cache. Pass it as CLI argument: `--dfx-cache-path=PATH`",
-        )
-    })?;
 
     match opts.subcmd {
-        SubCommand::Config(v) => commands::config::exec(v, dfx_cache_path),
-        SubCommand::Import(v) => commands::import::exec(v, dfx_cache_path),
-        SubCommand::Deploy(v) => commands::deploy::exec(v, dfx_cache_path),
-        SubCommand::Download(v) => commands::download::exec(v, dfx_cache_path),
-        SubCommand::PrepareCanisters(v) => commands::prepare_canisters::exec(v, dfx_cache_path),
-        SubCommand::Propose(v) => commands::propose::exec(v, dfx_cache_path),
+        SubCommand::DeployTestflight(args) => {
+            deploy_testflight(args);
+            Ok(())
```

Comment:
Does deploy_testflight return something? If so, this clobbers that. Is clobbering really what we want here?

Reply:
No, it returns `()` so there's not really anything to clobber

Reply:
Yes it does